main.go

```diff
@@ -123,22 +129,26 @@ func syncRequired() bool {
 // Starts the sync process only if it is not running
 func startSyncDispatcher() {
 	if syncRequired() && !syncDispatcherRunning {
-		sync := newSyncDispatcher(varnamdConfig.syncIntervalInSecs * time.Second)
+		sync := newSyncDispatcher(varnamdConfig.syncInterval * time.Second)
 		sync.start()
 		sync.runNow() // run one round of sync immediatly rather than waiting for the next interval to occur
+
 		syncDispatcherRunning = true
 	}
 }
 
 func main() {
-	runtime.GOMAXPROCS(runtime.NumCPU())
-	flag.Parse()
 	varnamdConfig = initConfig()
 	startedAt = time.Now()
+
+	runtime.GOMAXPROCS(runtime.NumCPU())
+	flag.Parse()
```

Comment:
This should be called before initConfig() is called, otherwise flag values won't be set

Reply:
I will check this today.